src/routes/commentsRoutes/authCommentsRoutes.js

```diff
@@ -1 +1,33 @@
-// TODO
+const { createComment, deleteComment, updateComment } = require('../../schemas/commentsSchema')
+
+module.exports = async function authCommentRoutes(fastify) {
+  fastify.post('/comments', { schema: createComment }, async request => {
+    const { content } = request.body
+    const client = await fastify.pg.connect()
+    const { rows } = await client.query('INSERT INTO comments (content) VALUES ($1)', [content])
+    client.release()
+    return { code: 201, message: 'Comment has been created.', rows }
```

Comment:
When you return rows, the query needs to have 'RETURNING *' when you're doing an insert, otherwise it'll be empty